BedrockServer.cpp

```diff
@@ -734,6 +734,7 @@ void BedrockServer::sync(const SData& args,
 
     // We're really done, store our flag so main() can be aware.
     server._syncThreadComplete.store(true);
+
```

Comment:
don't need this.

Reply:
Yeah...I mean to be fair, the "reset plugin" test should also be here, not in FuzzyBot...